.github/workflows/license-headers.yml

```
@@ -0,0 +1,16 @@
+# Copyright VMware, Inc.
+# SPDX-License-Identifier: APACHE-2.0
+
+---
+name: license-headers
```

We usually add a prefix to order the actions (i.e `[CI/CD] ...`, `[Support] ...`, etc) and a descriptive title

What do you think about using something like this?
```suggestion
name: [License] Check license headers
```